sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -466,19 +451,22 @@ export class ManagementClient extends LinkEntity {
 
   /**
    * Peeks the desired number of messages from the specified sequence number.
-   * @param {Long} fromSequenceNumber The sequence number from where to read the message.
-   * @param {number} messageCount The number of messages to retrieve. Default value `1`.
-   * @param {string} sessionId The sessionId from which messages need to be peeked.
+   *
+   * @param associatedLinkName The name of the associated receiver link
+   * @param fromSequenceNumber The sequence number from where to read the message.
+   * @param messageCount The number of messages to retrieve. Default value `1`.
+   * @param sessionId The sessionId from which messages need to be peeked.
    * @returns Promise<ReceivedMessageInfo[]>
    */
   async peekBySequenceNumber(
+    associatedLinkName: string | undefined,
```

Comment:
Explanation makes sense - it sounds like this should be in options then, right? Ie, it sounds truly optional.

Also, your explanation sounds great - we should toss that into a comment in the source.

Reply:
wait, which explanation?

Reply:
You and @chradek had a little comment thread on this about what the parameter represents, and how it's optional, etc...

Just seemed like useful information for someone coming along and wondering what the parameter actually does beyond just mechanics.